pkg/kv/fs_test.go

```diff
@@ -0,0 +1,50 @@
+package kv
+
+import (
+	"context"
+	"testing"
+
+	"go.uber.org/zap"
+)
+
+func TestSetGet(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	logger := zap.NewExample()
+	store := NewFSStore(logger, "testfs")
```

Comment:
I googled a bit. I think we can use `t.TempDir` here and, therefore, we don't need to clean up the test directory. by ourselves

ref https://pkg.go.dev/testing#T.TempDir